packages/components/src/SvgIcon/utils/createSvgIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { $Diff } from "utility-types";
+import SvgIcon from "../SvgIcon";
+import type { SvgIconProps } from "../SvgIcon";
+
+type IconProps = $Diff<
+  SvgIconProps,
+  {
+    children: any;
```

Comment:
Number 2 of 3 `any` types in this PR.

Reply:
Good call. `$Diff` confuses the heck out of me. Let's remove it, and any other Flow-isms.